### PR TITLE
docs: fixing broken links

### PR DESCRIPTION
**Pull request by phelewski**
This updates the linking for several items in the documentation. I noticed that several links pointed to non-existent items or 404s. I've gone through the pages on the left hand side panel and adjusted all that I could find. I did test this with the remote netlify site.

### CODE REVIEW

File: docs/content/contrib-development.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ git commit -s
 git push origin somefeature
```
Comment (srenatus) [inline docs/content/contrib-development.md:81]: Thanks for fixing these! As you've probably guessed, they're remnants from moving a markdown file that was only accessible via github into the website 😳

File: CHANGELOG.md [hunk 2]
```diff
@@ -1485,7 +1485,7 @@ more information see https://openpolicyagent.org/docs/latest/privacy/.
 #### New `opa build` command
 
 The `opa build` command can now be used to package OPA policy and data files
-into [bundles](https://www.openpolicyagent.org/docs/latest/management/#bundles)
+into [bundles](https://www.openpolicyagent.org/docs/bundles.html)
```
Comment (srenatus) [inline CHANGELOG.md:1488]: This one doesn't look quite right 🤔 This one would be redirected to `/docs/latest/bundles` via
https://github.com/open-policy-agent/opa/blob/main/docs/website/layouts/index.redirects#L33, which redirects to `/docs/latest/management-bundles`, so can we just link there directly?

File: docs/README.md [hunk 3]
```diff
@@ -96,7 +96,7 @@ This option provides the best preview of the site content, using the exact same
    user account (which should have been configured in the previous step).
    
 1) Netlify will then upload the built content and serve it via their CDN. A URL to the preview will be given in the
-   CLI console output.
+   CLI console output. Be sure to select the `edge` version within the Documentation pages to see your reflected changes in the remote site.
```
Comment (srenatus) [inline docs/README.md:99]: Thanks 👏

**Review by srenatus — COMMENTED** [review]
Thank you so much. One nitpick on the CHANGELOG.md, but that aside, this looks great. 😃

**Comment by srenatus** [comment]
Thanks again. Something here is slightly off, there's one commit by @anderseknert in the PR branch... Would you mind rebasing and squashing? We can get this merged then. 🚀

File: CHANGELOG.md [hunk 4]
```diff
@@ -1485,7 +1485,7 @@ more information see https://openpolicyagent.org/docs/latest/privacy/.
 #### New `opa build` command
 
 The `opa build` command can now be used to package OPA policy and data files
-into [bundles](https://www.openpolicyagent.org/docs/latest/management/#bundles)
+into [bundles](https://www.openpolicyagent.org/latest/management-bundles)
```
Comment (anderseknert) [inline CHANGELOG.md:1488]: The `docs` part should be kept here.

File: docs/README.md (hunk 3 above)
Comment (anderseknert) [reply docs/README.md:99]: Indeed! 😃

**Review by anderseknert — CHANGES_REQUESTED** [review]
This is a fantastic contribution! Thanks 👏 
One error in the first change, except for that LGTM!

**Review by anderseknert — APPROVED** [review]
LGTM! 👍

**Review by anderseknert — APPROVED** [review]

**Comment by anderseknert** [comment]
Thank you!

**Pull request merged**